src/transformers/models/convbert/configuration_convbert.py

```diff
@@ -138,3 +142,19 @@ def __init__(
         self.conv_kernel_size = conv_kernel_size
         self.num_groups = num_groups
         self.classifier_dropout = classifier_dropout
+
+
+class ConvBertOnnxConfig(OnnxConfig):
```

Comment:
Since this class is identical to `BertOnnxConfig`, I suggest adding a `# Copied from` statement so and changes in the original are caught in the copies:

```suggestion
# Copied from transformers.models.bert.configuration_bert.BertOnnxConfig
class ConvBertOnnxConfig(OnnxConfig):
```